index.js

```diff
@@ -48,6 +55,11 @@ console.log('hello world')
 
 // TODO: ADD CODE HERE
 
+const price = 4.99;
+const quantity = 15;
+const total = price * quantity;
+console.log(Math.round(total));
```

Comment:
The `total` should be rounded to the nearest two decimal points, not the nearest integer (`$74.85` instead of `75`)

Reply:
Oh, this is embarrassing! I should use total.toFixed(2), instead....
I will fix it.
Thank you!